runner-manager/cf-driver/prepare.sh

```diff
@@ -107,7 +106,13 @@ start_container () {
     container_command=$(echo "$img_data" | jq -r '.command | select(.)')
 
     if [ -n "$container_entrypoint" ] || [ -n "$container_command" ]; then
-        push_args+=('-c' "${container_entrypoint[@]}" "${container_command[@]}")
+        push_args+=('-c')
+        if [ -n "$container_entrypoint" ]; then
+            push_args+=("${container_entrypoint[@]}")
+        fi
+        if [ -n "$container_command" ]; then
+            push_args+=("${container_command[@]}")
+        fi
```

Comment:
[question (non-blocking)] Sorry! was it an empty string making trouble?

Reply:
yeah - I forget the exact error but it was about `""` not being valid I think when I supplied entrypoint but not command.